colectica_api/api.py

```diff
@@ -764,6 +764,53 @@ def get_a_set_lasted(self, AgencyId, Identifier):
             if response.json() != []:
                 return response.json()
 
+    def update_state(
+        self,
+        items,
+        *,
+        Version=None,
```

Comment:
Should this `Version` keyword argument be deleted?  It isn't documented and seems unused.